src/portal/ui/viewer/proc_par.cljs

```diff
@@ -0,0 +1,340 @@
+(ns portal.ui.viewer.proc-par
+  (:require ["dagre" :as dagre]
+            ["html-to-image" :as html-to-image]
+            ["react" :as react]
+            ["reactflow" :as flow
+             :default ReactFlow
+             :refer [Background Controls Handle ReactFlowProvider applyEdgeChanges Position applyNodeChanges ControlButton]]
```

Comment:
I'm not sure if `:default` is standard cljs, I usually see this pattern instead `:refer [default] :rename {default ReactFlow}`